### PR TITLE
Add page cookies

### PR DESCRIPTION
**Pull request by NicolasHov**

### CODE REVIEW

**Comment by HugoGresse** [comment]
Can you also add a screenshot of the page in desc?

**Comment by NicolasHov** [comment]
Screenshot (as stated in code, I didn't found a cookies icon in mui
Screenshot 2020-12-17 at 16 29 41

**Review by ntchou — APPROVED** [review]

File: src/pages/AcceptCookies/AcceptCookies.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+`
+
+export default function AcceptCookies() {
+    const { t } = useTranslation('accept-cookies')
+
+    return (
+        <IonContent style={{ '--background': 'white' }}>
+            <InformationBlock>
+                {/* Find a better Icon - didn't find cookies in Mui */}
```
Comment (HugoGresse) [inline src/pages/AcceptCookies/AcceptCookies.tsx:33]: you can use the unicode cookie: 🍪

# 🍪

**Comment by MattiaPrimavera** [comment]
I rebased on master + added French translations

**Review by ntchou — APPROVED** [review]

**Pull request merged**